core/resources/test_nested_multiplicities.xml

```diff
@@ -0,0 +1,53 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms"
+        xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+        xmlns:jr="http://openrosa.org/javarosa">
+    <h:head>
+        <h:title>Constraints for simple and select answers</h:title>
+        <model>
+            <instance>
+                <data xmlns:jrm="http://dev.commcarehq.org/jr/xforms"
+                      xmlns="http://openrosa.org/formdesigner/21D20D14-BF66-497D-8258-8356080CDB3C"
+                      uiVersion="1" version="279" name="simple structure to test nested multiplicity queries">
+                    <show>yes</show>
+                  <bikes>
+                    <manufacturer id="surly" american="yes">
+                      <model id="karate-monkey" color="black"/>
+                      <model id="long-haul" color="black"/>
+                      <model id="cross-check" color="silver"/>
+                    </manufacturer>
+                    <manufacturer id="moots" american="yes">
+                      <model id="vamoots" color="silver"/>
+                    </manufacturer>
+                    <manufacturer id="bianchi" american="no">
+                      <model id="pista" color="seafoam"/>
+                      <model id="zolder" color="silver"/>
+                    </manufacturer>
+                  </bikes>
+
+                  <silver-american>none</silver-american>
+                  <pista-color>blue</pista-color>
+                </data>
+            </instance>
+
+            <setvalue event="xforms-ready" ref="/data/silver-american"
```

Comment:
are we using these anymore or are we using the inline tests? Seems confusing to include them in both places?